docs/conf.py

```diff
@@ -38,28 +34,3 @@ class intergov:
 pygments_style = 'sphinx'
 html_theme = 'alabaster'  #'default'
 html_static_path = ['_static']
-htmlhelp_basename = 'intergov_ledgerdoc'
-
-latex_elements = {
-    'papersize': 'a4paper',
-}
-latex_documents = [
-    ('index', 'intergov_ledger.tex',
-     u'Intergov Ledger Documentation',
-     u'Chris Gough', 'manual'),
-]
-
-man_pages = [
-    ('index', 'intergov_ledger',
-     u'Intergov Ledger Documentation',
-     [u'Chris Gough'], 1)
-]
-
-texinfo_documents = [
-    ('index', 'intergov_ledger',
-     u'Intergov Ledger Documentation',
-     u'Chris Gough',
-     'intergov_ledger',
-     'One line description of project.',
-     'Miscellaneous'),
-]
```

Comment:
honestly, who throws a shoe!

(nobody uses info pages or man pages for this sort of thing, good deletion)